iot-edge/EdgeSolution/.env

```diff
@@ -0,0 +1,2 @@
+CONTAINER_REGISTRY_USERNAME=[USERNAME]
```

Comment:
This is a dangerous practice. Passwords should never be added to a file without any kind of encryption.